src/component/menu/Header/Header.tsx

```diff
@@ -114,9 +136,20 @@ const Header: VFC = () => {
                     )}
                     aria-label="Home"
                 >
-                    <UnleashLogo
-                        className={styles.logo}
-                        aria-label="Unleash logo"
+                    <ConditionallyRender
```

Comment:
Maybe we should have a helper component for this case:

```tsx
<ThemeMode 
    light={<UnleashLogo ... />} 
    dark={<UnleashLogo ... />} 
/>
```

Reply:
Great idea